services/comprehension/feedback-api-main/src/endpoint/endpoint.go

```diff
@@ -63,18 +63,19 @@ func AssembleUrls() ([api_count]string) {
 	opinion_domain := GetOpinionDomain()
 
 	var (
-		automl_api = fmt.Sprintf("%s/api/v1/evidence/feedback/automl.json", lms_domain)
-		plagiarism_api = fmt.Sprintf("%s/api/v1/evidence/feedback/plagiarism.json", lms_domain)
-		sentence_structure_regex_api = fmt.Sprintf("%s/api/v1/evidence/feedback/regex/rules-based-1.json", lms_domain)
-		post_topic_regex_api         = fmt.Sprintf("%s/api/v1/evidence/feedback/regex/rules-based-2.json", lms_domain)
-		typo_regex_api               = fmt.Sprintf("%s/api/v1/evidence/feedback/regex/rules-based-3.json", lms_domain)
-		spell_check_bing             = fmt.Sprintf("%s/api/v1/evidence/feedback/spelling.json", lms_domain)
-
-		grammar_check_api = "https://quill.spell.services/Quill/grammar/predict"
-		opinion_check_api = fmt.Sprintf("%s/", opinion_domain)
+		automl_api 										= fmt.Sprintf("%s/api/v1/evidence/feedback/automl.json", lms_domain)
```

Comment:
The formatting here seems inconsistent.

Reply:
Notice the line change. It is not consistent, but github's diffing UI makes it appear so. Check the raw file to see what I mean.

Reply:
![Screen Shot 2021-11-11 at 11 59 31 AM](https://user-images.githubusercontent.com/2057805/141338286-f53f820e-e4d2-43b7-925e-205b6db88958.png)

It looks weird in my IDE too.

Reply:
Fixed!